### PR TITLE
Documentation + tox + test suites for daft-06 and draft-07

### PR DESCRIPTION
**Pull request by Jokipii**
Changes:
- README.markdon changed to README.rst, so it can be used as PyPI description
- Little more text on README.rst
- LICENCE file added
- tox.ini added to automate testing. Starts from py34, because py33 is not valid tox target
- setuo.py long_description read from README.rst so PyPI will show that
- draft-06 test suit with correct ignores
- draft-07 test suit with correct ignores

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-e .
+pytest
+sphinx
```
Comment (horejsek) [inline test-requirements.txt]: Test requirements are already in the setup.py.

File: tests/test_json_schema_test_suits_draft4.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        'ecmascript-regex.json',
+    ]
+    ignore_tests = []
+
+    suite_dir_path = Path(suite_dir).resolve()
+    test_file_paths = sorted(set(suite_dir_path.glob("**/*.json")))
+
+    param_values, param_ids = resolve_param_values_and_ids(
+        test_file_paths, ignored_suite_files, ignore_tests
+    )
```
Comment (horejsek) [inline tests/test_json_schema_test_suits_draft4.py]: I think we could pass also dir directly, so it doesn't have to be copy-pasted many times.

File: test-requirements.txt (hunk 1 above)
Comment (Jokipii) [reply test-requirements.txt]: Originally I didn't get tox working without, but now It works... remove file and commit changes.

File: tests/test_json_schema_test_suits_draft4.py (hunk 2 above)
Comment (Jokipii) [reply tests/test_json_schema_test_suits_draft4.py]: True... always better to have less code. I will change that

**Pull request merged**